tools/deploy

```diff
@@ -21,6 +21,6 @@ pushd $(dirname ${secrets_yaml})
 awsudo $ADMIN_ARN sops --decrypt $(basename ${secrets_yaml}) > $tmp_decrypted
 
 popd
-helm upgrade --debug --wait --install ${deployment_name}-${environment} hub -f deployments/common/config/all.yaml -f deployments/${deployment_name}/config/common.yaml -f deployments/${deployment_name}/config/${environment}.yaml -f ${tmp_decrypted} --set-string jupyterhub.singleuser.image.name=${account_id}.dkr.ecr.us-east-1.amazonaws.com/${deployment_name}-user-image
+helm upgrade --debug --wait --install ${deployment_name}-${environment} hub -f deployments/common/config/all.yaml -f deployments/${deployment_name}/config/common.yaml -f deployments/${deployment_name}/config/${environment}.yaml -f ${tmp_decrypted} --set-string jupyterhub.singleuser.image.name=${account_id}.dkr.ecr.us-east-1.amazonaws.com/${deployment_name}-user-image --set global.safeToShowValues=true
```

Comment:
Should safeToShowValues be permanent?   Could comment it out as a hint

Reply:
Not sure.  I think I grabbed that from Jake.  It does display sensitive info.  Happy to remove it if you think that's the right thing to do.

Reply:
Removed.